cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/WorkloadClusterInventory.tsx

```diff
@@ -0,0 +1,36 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { CdsButton } from '@cds/react/button';
+import { CdsIcon } from '@cds/react/icon';
+
+const WorkloadClusterInventory: React.FC = () => {
```

Comment:
we are using **function** keyword  in other components. I think we should keep it consistent.

Reply:
Ah yes, this was a copy paste mistake, thanks for catching it